### ISSUE TITLE
API request failed [500]: unsupported charset "ISO-8859-1"

### ISSUE THREAD

**Issue by BlackmanCC** [abaplint/abaplint#1615]
When I run the sci check againt abaplint backend. I get the error:

API request failed [500]: unsupported charset "ISO-8859-1"

Our System is an 740 non-unicode system. 

Do you have ideas how I can fix this?

**Comment by larshp** [comment]
this is when calling the API provided by https://github.com/abaplint/abaplint-sci-server ?

**Comment by BlackmanCC** [comment]
yes, I am running the API in a docker containmer on our internal linux server.
